gfx/layers/wr/WebrenderPaintedLayer.cpp

```diff
@@ -65,7 +65,7 @@ WebRenderPaintedLayer::RenderLayer(wrstate* aWRState)
   }
   wr_dp_push_image(aWRState, toWrRect(rect), toWrRect(clip), NULL, key);
   Manager()->AddImageKeyForDiscard(key);
-  wr_pop_dl_builder(aWRState, bounds.x, bounds.y, bounds.width + bounds.x, bounds.height + bounds.y, &transform.components[0]);
+  wr_pop_dl_builder(aWRState, toWrRect(Rect()), toWrRect(bounds), &transform.components[0]);
```

Comment:
Yeah, I agree. This change was intended to preserve the existing values being passed through - before this patch the bounds values were being passed through to rust and used as the overflow parameter to the StackingContext, so it was still confusing. I will clean this up in future patches and make sure the right values are getting pushed through.